src/index.html

```diff
@@ -1,14 +1,76 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo-wrapper">
+        <a href="#" class="logo-link">
+          <img
+            src="images/logo.png"
+            alt="MOYO"
+            class="main-logo"
+          >
+        </a>
+      </div>
+      <nav class="navi">
+        <ul class="navi-list">
+          <li class="navi-item">
```

Comment:
navi ???
страйся називати класи по методології БЕМ
https://ru.bem.info/methodology/
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```

Reply:
Сделаю, просто это задание раньше БЄМ и я им не владел еще